### PR TITLE
v2.4.1 updates

### PR DESCRIPTION
**Pull request by datamattsson**
- Added/changed Alletra Storage MP where relevant
- Added nfsForeignStorageClass verbiage
- Added v2.4.1 release
- Minor tweaks

### CODE REVIEW

**Review by dileepds — DISMISSED** [review]
LGTM

**Review by dileepds — APPROVED** [review]
LGTM

**Review by dileepds — DISMISSED** [review]
LGTM

**Review by dileepds — DISMISSED** [review]
LGTM

File: docs/container_storage_provider/hpe_alletra_storage_mp/index.md [hunk 1]
```diff
@@ -1,29 +1,33 @@
 # Introduction
 
-The HPE Alletra 9000 and Primera and 3PAR Storage Container Storage Provider (CSP) for Kubernetes is part of the [HPE CSI Driver for Kubernetes](../../csi_driver/index.md). The CSP abstract the data management capabilities of the array for use by Kubernetes. 
+The HPE Alletra Storage MP, Alletra 9000 and Primera and 3PAR Storage Container Storage Provider (CSP) for Kubernetes is part of the [HPE CSI Driver for Kubernetes](../../csi_driver/index.md). The CSP abstract the data management capabilities of the array for use by Kubernetes.
```
Comment (madhav-bharadwaj) [inline docs/container_storage_provider/hpe_alletra_storage_mp/index.md:3]: Should we call out HPE Alletra Storage MP as HPE Alletra MP for block? Do we follow official  branding nomenclature in our documentation?

File: docs/container_storage_provider/hpe_alletra_storage_mp/index.md [hunk 2]
```diff
@@ -1,29 +1,33 @@
 # Introduction
 
-The HPE Alletra 9000 and Primera and 3PAR Storage Container Storage Provider (CSP) for Kubernetes is part of the [HPE CSI Driver for Kubernetes](../../csi_driver/index.md). The CSP abstract the data management capabilities of the array for use by Kubernetes. 
+The HPE Alletra Storage MP, Alletra 9000 and Primera and 3PAR Storage Container Storage Provider (CSP) for Kubernetes is part of the [HPE CSI Driver for Kubernetes](../../csi_driver/index.md). The CSP abstract the data management capabilities of the array for use by Kubernetes. 
+
+!!! note
+    The HPE CSI Driver for Kubernetes is only compatible with HPE Alletra Storage MP running with block services, such as HPE GreenLake for Block Storage.
```
Comment (madhav-bharadwaj) [inline docs/container_storage_provider/hpe_alletra_storage_mp/index.md:6]: I dint notice this. Please ignore my previous comment

**Review by dileepds — APPROVED** [review]

**Pull request merged**